Formula/tezos.rb

```diff
@@ -8,16 +8,17 @@ class << self
   desc "Platform for distributed consensus with meta-consensus capability"
   homepage "https://gitlab.com/tezos/tezos"
 
-  url "https://gitlab.com/tezos/tezos.git", :revision => "64c855499057fa84b41fcb9f3d7d72d4570db1a7", :branch => "mainnet", :shallow => false
-  version "006_PsCARTHA"
+  url "https://gitlab.com/tezos/tezos.git", :revision => "6b9f3bc3c90d99d4a2bd5cd26e6f3efbcb977090", :branch => "latest-release", :shallow => false
```

Comment:
is it necessary to specify branch if we have revision?
also, this revision corresponds to tag `v7.2`, can we use tag instead?